lib/parser/parse-tree.h

```diff
@@ -1208,24 +1208,28 @@ WRAPPER_CLASS(ArrayConstructor, AcSpec);
 // R1124 do-variable -> scalar-int-variable-name
 using DoVariable = Scalar<Integer<Name>>;
 
-template<typename A> struct LoopBounds {
+template<typename A, typename B> struct LoopBounds {
```

Comment:
Maybe use `DOVAR` and `EXPR` or something like that.

Reply:
OK. I changed them to `VAR` and `BOUND`.